Spigot-API-Patches/0157-Textual-Chat-Format.patch

```diff
@@ -0,0 +1,670 @@
+From 44e0062a8eebeebfc4723cd4263986396550fc15 Mon Sep 17 00:00:00 2001
+From: MiniDigger <admin@minidigger.me>
+Date: Fri, 28 Sep 2018 08:08:15 +0000
+Subject: [PATCH] Textual Chat Format
+
+
+diff --git a/src/main/java/co/aikar/timings/TimingsReportListener.java b/src/main/java/co/aikar/timings/TimingsReportListener.java
+index e7c389c0..f6e429ff 100644
+--- a/src/main/java/co/aikar/timings/TimingsReportListener.java
++++ b/src/main/java/co/aikar/timings/TimingsReportListener.java
+@@ -9,6 +9,7 @@ import org.bukkit.command.MessageCommandSender;
+ import org.bukkit.command.RemoteConsoleCommandSender;
+ 
+ import java.util.List;
++import java.util.Map;
+ 
+ @SuppressWarnings("WeakerAccess")
+ public class TimingsReportListener implements MessageCommandSender {
+@@ -58,6 +59,21 @@ public class TimingsReportListener implements MessageCommandSender {
+         senders.forEach((sender) -> sender.sendMessage(message));
+     }
+ 
++    @Override
++    public void sendEnhancedMessage(String enhancedMessage) {
```

Comment:
I don't like this name.

Reply:
sendRichMessage?

Reply:
I got enhanced from the original issue, rich works great too